src/components/overview-header/overview-header.js

```diff
@@ -109,30 +102,34 @@ class OverviewHeader extends React.PureComponent {
     });
 
     return (
-      <div className="scroll-hidden border-b border--gray-light prose mb24">
+      <div className="dr-ui--overview-header border-b border--darken10 prose mb24 pr60-mxl">
         <h1 className="mb6 txt-fancy">
           {props.title}
           {props.tag && this.buildTag(props)}
         </h1>
-        <div className="relative">
-          <div className="pr12-ml mr240-ml mr0">
+        <div className="flex-parent">
```

Comment:
What do you think about moving the `<h1>` inside the left-hand, text area? I noticed there might be some spacing funkiness between the image on the right and the text on the left: 

<img width="769" alt="image" src="https://user-images.githubusercontent.com/2365503/98710877-d6e74e80-2349-11eb-814c-ba683d685252.png">

👇 

<img width="751" alt="image" src="https://user-images.githubusercontent.com/2365503/98710845-cdf67d00-2349-11eb-87ca-ede3f920aaf3.png">

Reply:
Updated!